store/storetest/channel_store.go

```diff
@@ -1046,20 +1046,17 @@ func testChannelStoreGetChannels(t *testing.T, ss store.Store) {
 		t.Fatal("missing channel")
 	}
 
-	acresult := <-ss.Channel().GetAllChannelMembersForUser(m1.UserId, false, false)
-	ids := acresult.Data.(map[string]string)
+	ids, _ := ss.Channel().GetAllChannelMembersForUser(m1.UserId, false, false)
```

Comment:
nit: Would be nice to add here a `require.Nil(t, err)` to verify that a error doesn't happen. (and the same in the rest of the changes of this file).